kubernetes/alb_ingress/kubernetes.tf

```diff
@@ -44,14 +46,20 @@ resource "kubernetes_ingress" "public" {
   }
 
   spec {
-    rule {
-      http {
-        path {
-          path = "/*"
+    dynamic "rule" {
+      for_each = var.base_redirect_subdomain != "" ? ["redirect-base"] : []
```

Comment:
I don't see the `each` object used here. Why did you opt to use `for_each` instead of `count` here?

Reply:
`count` would be better, but i don't believe it works for `dynamic` blocks. instead i have to do this hacky thing where i iterate through an array of size 0 or 1 to simulate the same thing.

Reply:
Huh. Well, good to know. Thanks!